test/DotnetIsolatedE2ETests/Fixtures/FixtureHelpers.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E
+
+{
+    public static class FixtureHelpers
+    {
+        public static Process GetFuncHostProcess(bool enableAuth = false)
+        {
+            var funcProcess = new Process();
+            var rootDir = Path.GetFullPath(@"../../../../..");
+            var e2eAppBinPath = Path.Combine(rootDir, @"test/DotnetIsolatedE2EApps/MainApp/bin");
+            string? e2eHostJson = Directory.GetFiles(e2eAppBinPath, "host.json", SearchOption.AllDirectories).FirstOrDefault();
+
+            if (e2eHostJson == null)
+            {
+                throw new InvalidOperationException($"Could not find a built worker app under '{e2eAppBinPath}'");
+            }
+
+            var e2eAppPath = Path.GetDirectoryName(e2eHostJson);
+
+            var cliPath = Path.Combine(rootDir, @"test/DotnetIsolatedE2ETests/Azure.Functions.Cli/func");
```

Comment:
This seems to imply that the func.exe tool is checked into the repo, which I assume is not your intention. I recommend putting it into a temporary directory instead.

Reply:
func isn't checked into the repo, but running the build-e2e-test.ps1 script will install it into this location. Useful for several reasons - 
1) Durable "power users" often modify the Core Tools in the PATH, we want an unmodified copy running for these tests
2) Pipeline automation will have to take this step anyway, might as well do it in this script

Reply:
I can modify the install directory - but for the pipeline scenario it is most useful to have it inside the repo folder structure somewhere. Dotnet worker does it in the root.